app/controllers/hub/documents_controller.rb

```diff
@@ -10,10 +10,13 @@ class DocumentsController < ApplicationController
     layout "admin"
 
     def index
-      @sort_order = sort_order
-      @sort_column = sort_column
-      @documents = @documents.except(:order).order({ @sort_column => @sort_order })
-      @document = Document.new # used for form to upload documents
+      @documents = sorted_documents.active
```

Comment:
This is more of a design/UX question, but do we have any sense of whether a user would find it helpful to see the archived documents in the same list as the regular documents (for comparison purposes, locating a similar/previous version of a document, etc?) It seems like it would be just as simple to click on the docs folder and have that reload the page with the archived documents expanded below the active docs.